123_state_tasks.R

```diff
@@ -14,20 +14,34 @@ do_state_tasks <- function(oldest_active_sites, ...) {
       sprintf("get_site_data('1_fetch/tmp/inventory_%s.tsv', parameter)", task_name)
     }
   )
+  plot_step <- create_task_step(
+    step_name = 'plot',
+    target_name = function(task_name, ...) sprintf('3_visualize/out/timeseries_%s.png', task_name),
+    command = function(task_name, steps, ...) {
+      sprintf("plot_site_data(out_file = target_name, site_data = %s, parameter = parameter)", steps[['download']]$target_name)
+    }
+  )
+  tally_step <- create_task_step(
+    step_name = 'tally',
+    target_name = function(task_name, ...) sprintf('%s_tally', task_name),
+    command = function(task_name, steps, ...) {
+      sprintf("tally_site_obs(site_data = %s)", steps[['download']]$target_name)
+    }
+  )
 
   # Create the task plan
   task_plan <- create_task_plan(
     task_names = task_names,
-    task_steps = list(download_step),
+    task_steps = list(download_step, plot_step, tally_step),
     add_complete = FALSE)
 
   # Create the task remakefile
   create_task_makefile(
     task_plan = task_plan,
     makefile = '123_state_tasks.yml',
     include = 'remake.yml',
-    sources = '1_fetch/src/get_site_data.R',
-    packages = c('tidyverse', 'dataRetrieval'),
+    sources = c(...
```

Comment:
Do you need to close this with `c(...),`?

Reply:
whoops